src/app/api/product-reviews/[productId]/route.ts

```diff
@@ -0,0 +1,42 @@
+import { getProductReviews } from "@/model/ProductReviewsModel";
+import { NextResponse } from "next/server";
+
+export async function GET(
+  request: Request,
+  { params }: { params: { productId: string } }
+) {
+  const { productId } = params;
+
+  try {
+    let reviews = await getProductReviews(productId);
+    if (reviews !== undefined) {
+      return NextResponse.json(
+        {
+          reviews,
+        },
+        {
+          status: 200,
+        }
+      );
+    } else {
+      return NextResponse.json(
+        {
+          message: "There are no reviews",
```

Comment:
The message should be in singular.

> There is no review with id=...

Reply:
This function returns the Product Reviews of a product. The message should be in plural.